code/modules/clothing/clothing.dm

```diff
@@ -215,6 +215,8 @@ BLIND     // can't see anything
 	var/obj/item/weapon/cell/cell = 0
 	var/clipped = 0
 	var/fingerprint_chance = 0
+	var/obj/item/clothing/ring/ring = null		//Covered ring
+	var/mob/living/carbon/human/wearer = null	//Used for covered rings when dropping

```

Comment:
This is cleared on drop, or on destroy, correct?

Reply:
It is cleared on drop.